model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -187,8 +183,8 @@ def _update_best_measurement(self, measurement: RunConfigMeasurement):
                 self._best_coordinate = self._coordinate_to_measure
                 self._best_measurement = measurement
 
-    def _get_last_results(self) -> RunConfigMeasurement:
-        return self._neighborhood.coordinate_data.get_measurement(
+    def _get_last_results(self) -> Optional[RunConfigMeasurement]:
```

Comment:
Why is this now optional? Is it because it can now return None?

Reply:
It always could be None. Some code changes I did made the type checking unhappy.